lib/iris/coord_systems.py

```diff
@@ -292,37 +327,109 @@ def as_cartopy_projection(self):
         )
 
     def as_cartopy_globe(self):
-        return self._crs.globe
+        return self._globe
+
+    @cached_property
```

Comment:
How does cached_property work under the covers? This property depends on `self._datum`. It seems that `self._datum` could change but `self._globe` is still cached?